From d8cfaa8709f431d436e67ec7cc37680f8b6478c2 Mon Sep 17 00:00:00 2001
From: Riddhi Aggarwal <quic_riddagga@quicinc.com>
Date: Fri, 19 May 2023 16:01:02 +0530
Subject: [PATCH] arm64: dts: qcom: ipq9574: Adding build support for various
 RDPs

Adding build support for the following RDPs
RDP417
RDP433-emmc-mht

Change-Id: Id505edb27e391652260a9a39aa75eab009722e99
Signed-off-by: Riddhi Aggarwal <quic_riddagga@quicinc.com>
---
 arch/arm/boot/dts/Makefile        | 2 ++
 arch/arm64/boot/dts/qcom/Makefile | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index abbdc3adbc27..076a060ae149 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -1048,8 +1048,10 @@ dtb-$(CONFIG_ARCH_QCOM) += \
 	qcom-ipq4019-ap.dk07.1-c2.dtb \
 	qcom-ipq8064-ap148.dtb \
 	qcom-ipq8064-rb3011.dtb \
+	ipq9574-rdp417.dtb \
 	ipq9574-rdp418.dtb \
 	ipq9574-rdp433.dtb \
+	ipq9574-rdp433-mht-phy.dtb \
 	ipq9574-rdp449.dtb \
 	ipq9574-rdp453.dtb \
 	ipq9574-rdp454.dtb \
diff --git a/arch/arm64/boot/dts/qcom/Makefile b/arch/arm64/boot/dts/qcom/Makefile
index 770da39f991f..796682457fc0 100644
--- a/arch/arm64/boot/dts/qcom/Makefile
+++ b/arch/arm64/boot/dts/qcom/Makefile
@@ -12,8 +12,10 @@ dtb-$(CONFIG_ARCH_QCOM)	+= ipq6018-cp01-c1.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq8074-hk01.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq8074-hk10-c1.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq8074-hk10-c2.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp417.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp418.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp433.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp433-mht-phy.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp449.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp453.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp454.dtb
-- 
2.34.1

